stock-tracker/src/store/store.ts

```diff
@@ -12,9 +12,14 @@ const middleware = [
   chartMiddleware({ socketService })
 ];
 
-// only using any here cause it is a one off! Don't moan
-const composeEnhancers =
-  (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+// Module augmentation - Redux devtools extension compose exists in a dev environment when using Redux DevTools
```

Comment:
maybe remove the comment

Reply:
I like the comment :D 
I was suggested not to use any because then TS loses it's purpose, instead you can give redux devtools the same type as compose